core/LSU.cpp

```diff
@@ -10,43 +10,35 @@ namespace olympia
     // Constructor
     ////////////////////////////////////////////////////////////////////////////////
 
-    LSU::LSU(sparta::TreeNode *node, const LSUParameterSet *p) :
-        sparta::Unit(node),
-        memory_access_allocator(50, 30),   // 50 and 30 are arbitrary numbers here.  It can be tuned to an exact value.
-        load_store_info_allocator(50, 30),
-        ldst_inst_queue_("lsu_inst_queue", p->ldst_inst_queue_size, getClock()),
-        ldst_inst_queue_size_(p->ldst_inst_queue_size),
-
-        tlb_always_hit_(p->tlb_always_hit),
-        dl1_always_hit_(p->dl1_always_hit),
-
-        issue_latency_(p->issue_latency),
-        mmu_latency_(p->mmu_latency),
-        cache_latency_(p->cache_latency),
-        complete_latency_(p->complete_latency)
+    LSU::LSU(sparta::TreeNode *node, const LSUParameterSet *p) : sparta::Unit(node),
+                                                                 memory_access_allocator(50, 30), // 50 and 30 are arbitrary numbers here.  It can be tuned to an exact value.
+                                                                 load_store_info_allocator(50, 30),
+                                                                 ldst_inst_queue_("lsu_inst_queue", p->ldst_inst_queue_size, getClock()),
+                                                                 ldst_inst_queue_size_(p->ldst_inst_queue_size),
+
+                                                                 tlb_always_hit_(p->tlb_always_hit),
+                                                                 dl1_always_hit_(p->dl1_always_hit),
+
+                                                                 issue_latency_(p->issue_latency),
+                                                                 mmu_latency_(p->mmu_latency),
+                                                                 cache_latency_(p->cache_latency),
+                                                                 complete_latency_(p->complete_latency)
```

Comment:
Um... hate the spacing.  😆 

To be fair, we need to put together some coding standards for this project.

Reply:
Ah sorry, will fix. I had copied my code over from another branch, but didn't realize the spacing was different.

Reply:
Spacing fixed in [5212a4c](https://github.com/riscv-software-src/riscv-perf-model/pull/70/commits/5212a4c3274e758abca7bebf8ebc45a4ac210ca9)